docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -0,0 +1,126 @@
+========================
+OAuth 2.0 authentication
+========================
+
+Trino can be configured to enable OAuth 2.0 authentication over HTTPS for the
```

Comment:
is it over HTTPS or actually over TLS?

Reply:
Is Trino using any other protocol than HTTP which can be secured with TLS?